horizon/enforcer/api.py

```diff
@@ -444,7 +443,7 @@ async def is_allowed_url(
             context=query.context,
             sdk=query.sdk,
         )
-        return await is_allowed(request, allowed_query, x_permit_sdk_language)
```

Comment:
why remove this ?

Reply:
don't think we use this but anyway

Reply:
It's only removing the x_permit_sdk_language as it was not used anywhere